src/Todos.Service/Model/TodoListModel.cs

```diff
@@ -1,11 +1,15 @@
-﻿public class TodoListModel
+﻿using System.Text.Json.Serialization;
+
+public class TodoListModel
```

Comment:
Ta klasa służy jednocześnie do przekazania parametrów do dodawania listy oraz do zwracania listy użytkownikowi. Rozdzieliłbym to na dwie klasy:
* CreateTodoListRequest - tutaj mamy parametry tworzenia listy
* TodoListModel - to zwracamy gdy użytkownik poprosi o listę

Dzięki temu nie trzeba będzie używać atrybutów do określania podstawowego konstruktora